lib/megaLogger.js

```diff
@@ -132,23 +136,21 @@
             'LOG': '#000000'
         },
         'dateFormatter': function(d) {
-            return d.getFullYear() + "-" + d.getMonth() + "-" + d.getDate() + " " + d.getHours() + ":" + d.getMinutes() + ":" + d.getSeconds() + "." + d.getMilliseconds();
+            return d.toISOString();
```

Comment:
Here an ISO 8601 date was being mistakenly created, with a wrong month and hour digits with no preceding zeros.

So, just replacing that with `toISOString()` should do the same in a perfect way.